spec/avro_spec.rb

```diff
@@ -3,7 +3,9 @@
 describe AvroTurf do
   include FakeFS::SpecHelpers
 
-  let(:avro) { AvroTurf.new(schemas_path: "spec/schemas/") }
+  let(:log) { StringIO.new }
+  let(:logger) { Logger.new(log) }
+  let(:avro) { AvroTurf.new(schemas_path: "spec/schemas/", logger: logger) }
```

Comment:
Since you put reader schemas under spec/schemas/reader, maybe it'd make sense to put writer schemas under spec/schemas/writer?